sbnana/CAFAna/Analysis/Fit.cxx

```diff
@@ -191,6 +191,14 @@ namespace ana
       // Need to deal with parameters that are not fit values!
       SystShifts shift = pt.shift;
 
+      // Deal with systs that are not fit values
```

Comment:
I think I understand this, but it took me a while to get back up to speed.
If "Need to copy over syst values into the seed for this fit that were in the input syst shifts, but are not being fit for, and therefore not part of the seedPts list" is an accurate description, could you elaborate the comment to something like that?